tox.ini

```diff
@@ -11,8 +11,15 @@ envlist =
 skip_missing_interpreters = true
 
 [testenv]
-deps = pytest
+passenv = CIRCLECI CIRCLE_* CI_PULL_REQUEST COVERALLS_PARALLEL COVERALLS_REPO_TOKEN COVERALLS_SERVICE_NAME
```

Comment:
export `COVERALLS_PARALLEL`  when `coverage` executes. You don't need to setup on CircleCI.

Reply:
Only COVERALLS_REPO_TOKEN was needed.